games/q.yaml

```diff
@@ -1,3 +1,17 @@
+- name: Q-Gears
+  images:
+  - http://q-gears.sourceforge.net/ss.jpg
+  lang: C++
+  license:
+  - GPL2
+  development: halted
+  originals:
+  - Final Fantasy VII
```

Comment:
This game will need to be added in the originals

Reply:
Sorted